src/backend/shortcuts/shortcuts/shortcuts.ts

```diff
@@ -172,7 +175,8 @@ async function generateMacOsApp(gameInfo: GameInfo) {
 
   logInfo('Generating macOS shortcut', LogPrefix.Backend)
 
-  const appShortcut = shortcutFiles(gameInfo.title)[0]!
+  // shortcutFiles => [desktop, menu] on mac, we don't add desktop shortcut
+  const appShortcut = shortcutFiles(gameInfo.title)[1]!
```

Comment:
replaced this to use `[1]` instead of `[0]` so it's more clear that this is not meant to be a desktop shortcut